Source/GSXib5KeyedUnarchiver.m

```diff
@@ -3044,6 +3053,62 @@ - (id) decodeYPlacementForElement: (GSXibElement *)element
   return [self _decodePlacementForObject: obj];
 }
 
+- (id) _decodeOrientationForObject: (id)obj
+{
+  NSLayoutConstraintOrientation orientation = 0L;
+  if ([obj isEqualToString: @"vertical"])
+    {
+      orientation = NSLayoutConstraintOrientationVertical;
+    }
+  else if ([obj isEqualToString: @"horizontal"])
+    {
+      orientation = NSLayoutConstraintOrientationHorizontal;
+    }
+  return [NSNumber numberWithInteger: orientation];
+}
+
+- (id) decodeOrientationForElement: (GSXibElement *)element
+{
+  id obj = [element attributeForKey: @"orientation"];
+  return [self _decodeOrientationForObject: obj];
+}
+
+- (id) _decodeDistributionForObject: (id)obj
+{
+  NSStackViewDistribution d = 0L;
+  if ([obj isEqualToString: @"equalCentering"])
+    {
+      d = NSStackViewDistributionEqualCentering;
+    }
+  else if ([obj isEqualToString: @"equalSpacing"])
+    {
+      d = NSStackViewDistributionEqualCentering;
+    }
+  else if ([obj isEqualToString: @"fill"])
+    {
+      d = NSStackViewDistributionFill;
+    }
+  else if ([obj isEqualToString: @"fillEqually"])
+    {
+      d = NSStackViewDistributionFillEqually;
+    }
+  else if ([obj isEqualToString: @"fillProportionally"])
+    {
+      d = NSStackViewDistributionFillProportionally;
+    }
+  else if ([obj isEqualToString: @"gravityAreas"])
+    {
+      d = NSStackViewDistributionEqualCentering;
```

Comment:
Shouldn't this be NSStackViewDistributionGravityAreas?

Reply:
No, in the xib these are the values for that property.   The test for this, by the way, is at https://github.com/gcasa/NSStackView_test.